src/app/shared/comcol/comcol-page-logo/comcol-page-logo.component.scss

```diff
@@ -0,0 +1,4 @@
+.logo-container {
+  max-width: var(--ds-comcol-logo-max-width);
+  max-height: var(--ds-comcol-logo-max-height);
+}
```

Comment:
Hi @alexandrevryghem , thanks for your feedback.

May I ask you how you performed the tests?
I was not able to reproduce the problem on communities and collections with the current code, the image sizes result capped to 500px and I would like to know how to reproduce the error, so that I can properly check if the current or the proposed change are correctly working.

On my side I have tested the logo upload creating a new community or collection and would seem fine for both.
Thanks in advance for the clarification.

Reply:
This is the test image I used:
<img src="https://github.com/DSpace/dspace-angular/assets/43750381/a693bfb6-f54c-4d8f-a086-80849817f31e" width="50%"/>
Results in this:
<img src="https://github.com/DSpace/dspace-angular/assets/43750381/b1cb4e0a-87c4-4260-b1e7-4ec99b9c46f3" width="50%"/>

Reply:
Hi @alexandrevryghem , thanks again for the feedback and the test scenario.
I ran some tests and actually the image you provided is rendered out of the container.
I have adapted the solution setting the max sizes directly on the img as you suggested, I've just omitted the auto values since they are the default and not necessary.

Reply:
Hi @FrancescoMolinaro, it's almost good. I also tested it in mobile mode today and I saw that the image overflowed out of the page this can easily be fixed by applying a `width: 100%;` on the `img`.

<img width="50%" src="https://github.com/DSpace/dspace-angular/assets/43750381/aa76c113-6c97-462d-b575-9f57a8a64733">